Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -1688,7 +1753,8 @@ def _buildClasses():
 		if m:
 			# XxxFormatN subtable, we only add the "base" table
 			name = m.group(1)
-			baseClass = FormatSwitchingBaseTable
+			formatType = table[0][0]  # can be 'uint16' or 'uint8'
```

Comment:
Can we use named access or add a comment to explain what table[0] and table[0][0] are meant to match?